nc_time_axis/tests/integration/test_plot.py

```diff
@@ -11,6 +11,11 @@
 import cftime  # nopep8
 import numpy as np  # nopep8
 
+try:
+    from cftime import datetime_base as cftime_datetime
+except ImportError:
+    from cftime import datetime as cftime_datetime
+
```

Comment:
This would be better sanitised by actually checking the `cftime` version to determine the correct behaviour (using [packaging.version.parse](https://packaging.pypa.io/en/latest/version/#packaging.version.parse) or similar).